install/kubernetes/helm/istio/values.yaml

```diff
@@ -180,6 +180,11 @@ global:
   # galley.enabled=true`).
   useMixerMCP: false
 
+  # Include the helm test templates when generating the template.
+  # If set to false, 'helm template ...' will ignore the helm test yaml files
+  # when generating the template
+  enabledTest: true
```

Comment:
I prefer we use the name as `enableHelmTest `

Reply:
done